support/tech-resources/enabling-https-engagement-tracking-on-sparkpost.md

```diff
@@ -105,93 +119,169 @@ Note: If you would like this tracking domain to be the default, please add `"def
 
 10. Navigate to the Tracking Domains section in the UI and click the orange "test" verification link. At this point, the process is complete.
 
-## Step by Step Guide with AWS CloudFront	
 
-Note: If you utilize CloudFront as your CDN to manage certificates and keys for any custom engagement tracking domains, it will result in a loss of user agent data. Please follow the “Updating an Existing Domain on AWS CloudFront” steps to minimize data loss.
+---
+## <a name="aws-create"></a> Step by Step Guide with AWS CloudFront
+
+Note: If you utilize CloudFront as your CDN to manage certificates and keys for any custom engagement tracking domains, it will result in a loss of user agent data. We include steps below to minimize data loss.
+
+The following is a sample guide for use with AWS CloudFront **only**; please note, the steps to configure your chosen CDN will likely differ from CloudFront in workflow. Please refer to your CDN's documentation and contact their respective support departments if you have any questions.
+For up to date information on creating a distribution via CloudFront, please refer to the [AWS docs](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/distribution-web-creating-console.html).
+
+1. Login with your credentials onto AWS console and navigate to [CloudFront](https://console.aws.amazon.com/cloudfront/).
+
+1. Choose **Create Distribution**:
+
+    ![](media/enabling-https-engagement-tracking-on-sparkpost/cloudfront_UI.png)
+
+1. On the "Select a delivery method for your content" page, under the **Web** section, choose **Get Started**.
+
+    ![](media/enabling-https-engagement-tracking-on-sparkpost/cloudfront_method.png)
+
+1. On the Create Distribution page fill out the following:
+    * Under Origin Settings, fill in the **Origin Domain Name**. Use the value displayed in the tracking domains section of the SparkPost UI. E.g.: for SparkPost US, this would be `spgo.io`; for SparkPost EU, this would be `eu.spgo.io`.
```

Comment:
PMTA is different if for some reason customers want to use a CDN there.  We should add a note for PMTA customers to see their own docs or add pmta.spgo.io or pmta.eu.spgo.io here.

Reply:
Good catch.